node/utils/metrics/metrics.ts

```diff
@@ -1,6 +1,7 @@
 import axios from 'axios'
 
-const ANALYTICS_URL = 'https://rc.vtex.com/api/analytics/schemaless-events'
+const ANALYTICS_URL =
+  'https://analytics.vtex.com/api/analytics/schemaless-events'
```

Comment:
why did this change? What is the difference between `rc` and `analytics` urls?

Reply:
I found [some issues](https://vtex.slack.com/archives/C02T3NYGVNX/p1699469093595349) to use `rc` and the [documentation](https://github.com/vtex/internal-docs/blob/main/docs/Analytics/VTEX-Data-Platform/data-ingestion-API.md#schemaless-events-api) suggest to use `analytics`.

"This can be done by sending events to the following endpoint: https://analytics.vtex.com/api/analytics/schemaless-events (For applications on the VTEX Network) or to https://rc.vtex.com/api/analytics/schemaless-events (For applications outside the VTEX Network, such as Front End Applications)"